app/models/movie.rb

```diff
@@ -1,4 +1,29 @@
 class Movie < ApplicationRecord
+  before_save :capitalize_attributes
   validates :title, presence: true, uniqueness: true
-  validates :description, presence: true
+  validates :description, :release_date, :genre, presence: true
+  validate :attachment_validations
+
+  has_one_attached :thumbnail
+  has_one_attached :trailer
+  has_many_attached :posters
+
+  private
+  def capitalize_attributes
+    self.title.capitalize!
+    self.genre.capitalize!
+  end
+
+  def attachment_validations
+    if !thumbnail.attached?
+      errors.add(:thumnbail, 'must be present')
+    elsif !posters.attached?
+      errors.add(:posters, 'must be present')
+    elsif !trailer.attached?
+      errors.add(:trailer, 'must be present')
```

Comment:
Check if we can use `validates :thumnbail, : posters, :trailer, presence: true` instead of this method?

Reply:
sir i first tried that and it didn't work so i used this method.